cmd/openshift-install/gather.go

```diff
@@ -136,17 +136,17 @@ func runGatherBootstrapCmd(directory string) (string, error) {
 		return "", errors.Wrapf(err, "failed to fetch %s", config.Name())
 	}
 
-	tfstate, err := terraform.ReadState(tfStateFilePath)
+	tfstate, err := terraform.ReadState(tfBootstrapStateFilePath)
 	if err != nil {
-		return "", errors.Wrapf(err, "failed to read state from %q", tfStateFilePath)
+		return "", errors.Wrapf(err, "failed to read state from %q", tfBootstrapStateFilePath)
 	}
 	bootstrap, port, masters, err := extractHostAddresses(config.Config, tfstate)
```

Comment:
Does this now need to be modified to get the IP addresses from both tf state files? The information is now split between two files.